tools/check_format.py

```diff
@@ -77,8 +79,8 @@ def whitelistedForProtobufDeps(file_path):
 # Production time sources should not be instantiated in the source, except for a few
```

Comment:
Real clock time sources should not be...

Reply:
done